timemachine/potentials/generic.py

```diff
@@ -15,8 +15,7 @@
 Conf = Array
 Params = Array
 Box = Array
-Lambda = float
-PotentialFxn = Callable[[Conf, Params, Box, Lambda], float]
+PotentialFxn = Callable[[Conf, Params, Box], float]
```

Comment:
nit completeness --- Also remove unused `lamb` references in `rmsd.py`?

Reply:
Removed in 004956e3b82be195357150b60dd8697ff9a2d91c